### PR TITLE
Push User Generated View with appdata

### PR DESCRIPTION
**Pull request by bh2smith**
To try this, create a new query in dune and run

```
SELECT * FROM dune_user_generated.gpv2_appdata
```

Note that the query ID here was published to my user account (so the query ID isn't aligned with the current user account for this project).

### CODE REVIEW

File: dune_api_scripts/store_query_result_all_distinct_app_data.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
     dune = dune_from_environment()
 
     # fetch query result id using query id
-    query_id = int(os.getenv('QUERY_ID_ALL_APP_DATA', "142824"))
+    query_id = int(os.getenv('QUERY_ID_ALL_APP_DATA', "257770"))
```
Comment (josojo) [inline dune_api_scripts/store_query_result_all_distinct_app_data.py:18]: Is this intended?

File: dune_api_scripts/update_appdata_view.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    # initialize the environment
+    dune = dune_from_environment()
+
+    # build query
+    QUERY = f"""CREATE OR REPLACE VIEW
+    dune_user_generated.gp_appdata (app_data, referrer)
+    AS VALUES {build_string_for_affiliate_referrals_pairs()}"""
+
+    # update query in dune
+    QUERY_ID = 257782
```
Comment (josojo) [inline dune_api_scripts/update_appdata_view.py:14]: I think its best to get it via an env variable, like here:
`    query_id = int(os.getenv('QUERY_ID_ALL_APP_DATA', "257770"))`

**Review by josojo — APPROVED** [review]
Cool

File: dune_api_scripts/store_query_result_all_distinct_app_data.py (hunk 1 above)
Comment (bh2smith) [reply dune_api_scripts/store_query_result_all_distinct_app_data.py:18]: No totally not!

File: dune_api_scripts/update_appdata_view.py (hunk 2 above)
Comment (bh2smith) [reply dune_api_scripts/update_appdata_view.py:14]: We also may need to settle on a single user account.

**Pull request merged**